src/main/java/com/teamwizardry/refraction/common/effect/EffectPlace.java

```diff
@@ -8,19 +8,32 @@
 import net.minecraft.entity.EntityLivingBase;
 import net.minecraft.entity.item.EntityItem;
 import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.item.Item;
 import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemBlockSpecial;
+import net.minecraft.item.ItemHoe;
+import net.minecraft.item.ItemSeeds;
```

Comment:
remove unused imports

Reply:
I think it will avoid unnecessary casting?